Slim/Control/Queries.pm

```diff
@@ -905,21 +884,36 @@ sub albumsQuery {
 					$contributorSth->bind_param(":work", $work);
 					$contributorSth->bind_param(":performance", $c->{'tracks.performance'}||undef);
 				}
-				$contributorSth->execute();
+				my $contributorArray = $dbh->selectall_arrayref($contributorSth,{ Slice => {} });
```

Comment:
This returns the SQL results as (or example):
```
$contributorArray=[
  { id => 530_486, name => "Gert Mortensen", role => 1 },
  { id => 530_481, name => "Kjell-Ake Pettersson", role => 1 },
  { id => 530_483, name => "Malm\xC3\xB6 Symphony Orchestra", role => 1 },
  { id => 530_482, name => "Per Falck", role => 1 },
  { id => 530_485, name => "Okko Kamu", role => 3 },
  { id => 530_483, name => "Malm\xC3\xB6 Symphony Orchestra", role => 4 },
]
```

Reply:
I understand we do this for each album individually? We don't risk to have one massive array in memory. Then `selectall_arrayref` should be good enough.

Reply:
I'm not sure what you are asking for here?

Reply:
Just confirmation that we do get the `$contributorArray` for each album individually. Just to make sure we don't try to grab a list of potentially tens of thousands of records in memory.